packages/core/test/Cardano/types/Block.test.ts

```diff
@@ -1,7 +1,12 @@
-import { BlockId } from '../../../src/Cardano';
+import { BlockId, VrfVkBech32 } from '../../../src/Cardano';
 
 describe('Cardano/types/Block', () => {
   it('BlockId() accepts a valid transaction hash', () => {
     expect(() => BlockId('0dbe461fb5f981c0d01615332b8666340eb1a692b3034f46bcb5f5ea4172b2ed')).not.toThrow();
   });
+
+  it('VrfVkBech32() accepts a valid vrf verification key bech32 string with vrf_vk or poolmd_vk prefix', () => {
```

Comment:
Missing a couple of throwing cases: 
1. invalid prefix
2. valid prefix, but truncated string